make-sync-function

```diff
@@ -42,30 +41,31 @@ try {
   // which it will be injected
   syncDocDefn = syncDocDefn.trim().replace(/(?:\r\n)|(?:\r)|(?:\n)/g, function() { return '\n  '; });
 } catch (ex) {
-  console.log('Unable to read the sync document definitions file: ' + ex);
-
-  return errorStatus;
+  if (ex.code === 'ENOENT') {
+    console.log('ERROR: Sync document definitions file does not exist');
```

Comment:
Because this error is fairly likely to occur in the wild, give it special handling to print a friendlier error message.